packages/site-components/src/NavigationItem/NavigationItem.tsx

```diff
@@ -7,7 +7,8 @@ import { ExpansionIcon } from './ExpansionIcon';
 import { useWindow } from '@salt-ds/window';
 import { useComponentCssInjection } from '@salt-ds/styles';
 
-import navigationItemCss from '@salt-ds/core/dist-es/navigation-item/NavigationItem.css.js';
+const navigationItemCss =
```

Comment:
Had to inline the CSS, this worked because core was bundled into site-components, but that broke using other components. Adding core to the package.json and inlining this was the easiest way to fix it.